cmd/inspr/cli/get.go

```diff
@@ -91,10 +91,10 @@ func getChannels(_ context.Context) error {
 	return nil
 }
 
-func getCTypes(_ context.Context) error {
+func gettypes(_ context.Context) error {
```

Comment:
I think it's better to change this func name to camel case, to maintain the project pattern